nixos/modules/services/desktops/geoclue2.nix

```diff
@@ -4,6 +4,10 @@
 
 with lib;
 
+let
+  # the demo agent isn't built by default, but we need it here
+  package = pkgs.geoclue2.override { withDemoAgent = true; };
```

Comment:
Can it be set to `config.services.geoclue2.enableDemoAgent`?

Reply:
Oh sure.

Reply:
Done.